### DISCUSSION TITLE
Make stealing media keys optional

### DISCUSSION THREAD

**Discussion by digitalfrost84** [f1multiviewer/issue-tracker#193] [Feature requests] [answered]
Once MultiViewer for F1 is on, it will capture all media keys. That means I cannot longer control my audio player via media keys like I'm used to.

Please make this an optional feature.

**Comment by f1multiviewer** [comment, marked as answer]
This is now available as an experimental setting in v1.7.0

**Reply by digitalfrost84** [reply]
Thanks that was fast. Works as expected for me.

**Reply by f1multiviewer** [reply]
Yeah I looked at it and seemed a simple fix, thanks for confirming it works!
